pkg/apis/build/v1alpha2/builder_types.go

```diff
@@ -65,6 +71,8 @@ type BuilderStatus struct {
 	ObservedStackGeneration int64                              `json:"observedStackGeneration,omitempty"`
 	ObservedStoreGeneration int64                              `json:"observedStoreGeneration,omitempty"`
 	OS                      string                             `json:"os,omitempty"`
+	Signed                  bool                               `json:"signed"`
+	SignaturePaths          []SignaturePath                    `json:"signaturePaths,omitempty"`
```

Comment:
I'm curious to the point of this bool, wouldn't the existence/non-emptiness of the `signaturePaths` field indicated if it's signed?

Reply:
Originally I thought about having it to indicate whether it has been signed or not but I agree that the `signaturePaths` field is enough. Will remove the `signed` field.

Reply:
Removed `Signed` as requested.